src/app/public/modules/autocomplete/autocomplete-default-search-function.ts

```diff
@@ -44,15 +44,6 @@ export function skyAutocompleteDefaultSearchFunction(
     const results = [];
 
     for (let i = 0, n = filteredData.length; i < n; i++) {
-      const limitReached = (
-        options.searchResultsLimit &&
-        options.searchResultsLimit <= results.length
-      );
-
-      if (limitReached) {
-        return results;
-      }
```

Comment:
Ya. That is probably a good catch. The background here is that this was causing issues where we were limiting the search results being returned instead of limiting what was being shown from those results. This caused issues when we add a button that should be showing _all_ search results as we didn't have them. 

I look at this though and we can probably deprecate the `searchResultsLimit` and just change the default to not exist instead of defaulting to 5 which is what I believe it was in the past.

Reply:
I added this back but did not add back where we were using this internally as we are handling this in the autocomplete template instead of this function now. This is because we need to have all search results but only show some of them. 

I'm not sure why we are exporting the options in the public API as it isn't documented, but since it is exported it is best that we don't remove a property.

Reply:
I’d rather do one of two things: 1) Remove the default function from our public API and make the necessary changes (since it's internal, and an audit of the functions usage turned up zero results), or 2) Keep it in the API and write unit tests for the existing “limit” logic. I’m personally okay with option 1 if you don’t see any red flags.